glhe/properties/fluid.py

```diff
@@ -14,7 +14,7 @@ class Fluid(object):
 
     def __init__(self, inputs):
         self._fluid_name = inputs["type"].upper()
-        self._concentration = inputs["concentration"] / 100
+        self._concentration = inputs["concentration"] / 100.0
```

Comment:
A unit test was failing because this was doing integer math and coming up with zero.  The unit test expected a property string with "0.5" in it, and it was giving "0".

Reply:
👍